bot/quest.py

```diff
@@ -0,0 +1,411 @@
+import asyncio
+
+from aiogram import Bot, Dispatcher, types, Router, F
+from aiogram.filters import Command
+from aiogram.types import Message
+from aiogram.types import FSInputFile
+import httplib2
+from aiogram.fsm.context import FSMContext
+from aiogram.fsm.state import State, StatesGroup
+from aiogram.fsm.storage.memory import MemoryStorage
+from aiogram.filters import Command
+from googleapiclient.discovery import build
+from oauth2client.service_account import ServiceAccountCredentials
+import creds
+import logging
+import re
+from datetime import datetime
+
+BOT_TOKEN = '6430127705:AAEPvaAi2Z2Z7Epwop7DiGVvEpmuOQ8oGu8'

```

Comment:
DO NOT PUSH `BOT_TOKEN` TO OUR **PUBLIC** REPOSITORY. CLEAN IT FROM YOUR CODE AND I WILL REFRESH OUR TOKEN. YOU WILL GET IT IN A MESSAGE

HERE HOW YOU SHOULD DO IT
```suggestion
BOT_TOKEN = os.getenv('BOT_TOKEN')
```

Reply:
Fixed in the commit cafdea8